mros1_reasoner/scripts/mros1_reasoner.py

```diff
@@ -271,6 +272,8 @@ def timer_cb(event):
             # Adaptation feedback:
             if result == 1: # reconfiguration executed ok
                 rospy.logerr("= RECONFIGURATION SUCCEEDED =") # for DEBUGGING in csv
+                ## Set new grounded_configuration
+                grounded_configuration = str(fd.name)
```

Comment:
I thought I had fixed this problem in line 282 below, where I store the `fg` based on the name of the new `fd`. I am not using the `grounded_configuration` variable but on the initialization.
@marioney

Reply:
But that is still being used on https://github.com/rosin-project/mros1_reasoner/blob/d75385fff1bb8bc8c600314bc2c50acde2388600/mros1_reasoner/scripts/mros1_reasoner.py#L160

That's why I added it there